tests/app.rs

```diff
@@ -1,24 +1,23 @@
-use wasm_bindgen_test::{wasm_bindgen_test_configure, wasm_bindgen_test};
 use futures::prelude::*;
 use wasm_bindgen::JsValue;
 use wasm_bindgen_futures::JsFuture;
+use wasm_bindgen_test::{wasm_bindgen_test, wasm_bindgen_test_configure};
 
 wasm_bindgen_test_configure!(run_in_browser);
 
-
 // This runs a unit test in native Rust, so it can only use Rust APIs.
 #[test]
 fn rust_test() {
     assert_eq!(1, 1);
 }
 
-
 // This runs a unit test in the browser, so it can use browser APIs.
 #[wasm_bindgen_test]
 fn web_test() {
     assert_eq!(1, 1);
 }
 
+/*
```

Comment:
Why did you comment this out? I'd prefer if you deleted it entirely, but why did you do it in the first place?

Reply:
weird compile errors
dunno what half of this autogenerated stuff is anyway, looks totally unnecessary